internal/pkg/build/build.go

```diff
@@ -195,6 +199,11 @@ func (b *Build) Full() error {
 			}
 		} else {
 			// regular build or force, start build from scratch
+			if b.Conf.ImgCache == nil {
+				fmt.Println("WARNING!!! imgCache is undefined")
+				os.Exit(42)
```

Comment:
Given I saw this above I assume these are debugging statements. I think an error is appropriate here.